pkg/install/stack/kubermatic-master/default_application_catalog.go

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
EE code has to live in pkg/ee/. Just like the main_ce/ee, please create a wrappers_ce/ee and either call code in pkg/ee/ or do a NOP.